src/Models/Services/Utils.ts

```diff
@@ -921,7 +921,7 @@ export const getUrlFromString = (urlString: string): URL => {
 /** Checking if a given ADX cluster url is a safe url following a certain regex and hostname */
 export const isValidADXClusterUrl = (clusterUrl: string): boolean => {
     const isValidADXClusterHostUrl = (urlPrefix) =>
-        /^[a-zA-Z0-9]{4,22}.[a-zA-Z0-9]{1,}\b$/.test(urlPrefix);
+        /^[a-z][a-z0-9-]{3,21}.[a-zA-Z0-9]{1,}\b$/.test(urlPrefix);
```

Comment:
Following the docs saying "Cluster name must begin with a letter and contain between 4 and 22 lowercase alphanumeric characters." and it includes dashes even though it is not documented.

Reply:
In other places, we have used this regex, should we converge them?
const isValidAdtHostUrl = urlPrefix => /^[a-zA-z0-9]{1}[a-zA-Z0-9-]{1,60}[a-zA-Z0-9]{1}(\.api)\.[a-zA-Z0-9]{1,}$/.test(urlPrefix);